domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -740,7 +740,7 @@ constructor(
 
     private fun getResolutionSelector(aspectRatio: AspectRatio): ResolutionSelector {
         val aspectRatioStrategy = when (aspectRatio) {
-            AspectRatio.THREE_FOUR -> AspectRatioStrategy.RATIO_16_9_FALLBACK_AUTO_STRATEGY
+            AspectRatio.THREE_FOUR -> AspectRatioStrategy.RATIO_4_3_FALLBACK_AUTO_STRATEGY
             AspectRatio.NINE_SIXTEEN -> AspectRatioStrategy.RATIO_16_9_FALLBACK_AUTO_STRATEGY
             else -> AspectRatioStrategy.RATIO_16_9_FALLBACK_AUTO_STRATEGY
```

Comment:
For other aspect ratios, we should also use the `RATIO_4_3_FALLBACK_AUTO_STRATEGY`. This will ensure we get the maximum FOV for 1:1.